x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -151,8 +159,10 @@ export class HeadlessChromiumDriverFactory {
 
       logger.debug(`Browser page driver created`);
 
+      let isClosed = false;
       const childProcess = {
         async kill() {
+          if (isClosed) return;
```

Comment:
```suggestion
          if (page.isClosed()) return;
```

Reply:
This scenario of returning early is expected to happen once per each execution, right? Because there is the explicit closing of the browser, and the fallback cleanup that does it again, right?

Reply:
That's right.